ert/data/record/_record.py

```diff
@@ -354,19 +335,21 @@ async def load_collection_from_file(
 ) -> RecordCollection:
     if mime == "application/octet-stream":
         if is_directory:
+            tar_transformation = ert.data.TarRecordTransformation()
             return RecordCollection(
-                records=(BlobRecord(data=await make_tar(file_path)),),
+                records=(await tar_transformation.transform_output(mime, file_path),),
```

Comment:
Since tar_transformation is not used after you might do just:
ert.data.TarRecordTransformation.transform_output(...)